GPT/gpt.py

```diff
@@ -61,6 +61,20 @@ def gpt_answer_question(text_to_process: str) -> str:
         """
         return gpt_query(prompt, text_to_process)
 
+    def gpt_smart_clipboard(clipboard_text: str, destination_text: str):
```

Comment:
I think this could be renamed `gpt_smart_combine` or something analogous just to be super clear since it isn't really using the clipboard here, even if the prompt refers to it.